doc/transformations.rst

```diff
@@ -119,6 +119,16 @@ can be found in the API-specific sections).
 	  :ref:`MoveTrans <sec_move_trans>` transformation may be used
           for this.
 
+Kernels
+-------
+
+Occasionally it is not sufficient to transform only the generated
```

Comment:
I don't necessarily think that it is going to only be occasionally in the future so I would be a bit more bullish here and say something like ...
"PSyclone supports the transformation of Kernels as well as PSy-layer code. Currently ..."

Reply:
Done.